src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -56,11 +57,12 @@ const STEPS_DATA = [
   },
 ]
 
-const FAQ_DATA = [
+const FAQ_DATA = (chainId: number | undefined) => [
```

Comment:
nit: why not passing directly the string? seems cleaner logic

Reply:
I think this is more flexible just in case there are some other changes in text based on chainId in the future.